CsvParser/csvParser.go

```diff
@@ -172,7 +174,7 @@ func dynamicallyMakeDir(originPath string, locationCode string) string {
 	return originPath
 }
 
-//uses all the helper functions above to dynamically store incidents in a file tree hierachy
+// Uses all the helper functions above to dynamically store incidents in a file tree hierachy
 func placeIncidentInFileStruct(originPath string, incMap map[string]map[string][]incident.IncidentData) {
```

Comment:
This function name makes me think that there is a struct involved. What about the name "addIncidentToHierarchy" ?